test/js/ez-platform-ui-app.js

```diff
@@ -303,6 +303,25 @@ describe('ez-platform-ui-app', function() {
         });
     });
 
+    describe('handle `ez:navigateTo`', function () {
+        function fireNavigateTo () {
```

Comment:
what's the point of the function, it's used only once below in a very short test